packages/linux-kernel/Jenkinsfile

```diff
@@ -69,6 +69,11 @@ def pkgList = [
     ['name': 'ovpn-dco','scmCommit': 'v0.2.20231117',
      'scmUrl': 'https://github.com/OpenVPN/ovpn-dco',
      'buildCmd': 'cd ..; ./build-openvpn-dco.sh'],
+
+    // RTSP netfilter helper
+    ['name': 'nat-rtsp', 'scmCommit': '475af0a',
+     'scmUrl': 'https://github.com/maru-sama/rtsp-linux.git',
```

Comment:
A concern that repo is not developing now. The latest commit was 4 years ago. There are 2 PR without any response from the author several years.
IMHO it’s a bad idea to add “dead” repo.

Reply:
That repository is the upstream that both [Debian](https://packages.debian.org/bookworm/nat-rtsp-dkms) and [OpenWrt](https://github.com/openwrt/packages/blob/master/net/xtables-addons/patches/100-add-rtsp-conntrack.patch) recognize. For OpenWrt, they have converted it into a patch which builds it as an in-tree module, while Debian is using DKMS. Neither of these approaches is a good fit for VyOS in my opinion.

The module has been in the wild in one form or other for past two decades and this repository has a track record of updating support for newer kernels whenever there has been a need for it. In terms of both age and update frequency, you can find similar examples from the [kernel tree](https://github.com/torvalds/linux/tree/master/net/netfilter) itself.

I cannot comment though why that module has never found its way into the kernel, it's definitely one of the simpler conntrack modules out of the bunch, however lack of inclusion to mainline could be cause for concern as it has not been exposed to as wide of an audience to scrutinize.